packages/notte-sdk/src/notte_sdk/types.py

```diff
@@ -441,6 +441,52 @@ class PersonaCreateResponse(BaseModel):
     persona_id: Annotated[str, Field(description="ID of the created persona")]
 
 
+class VaultCreateRequestDict(TypedDict, total=False):
```

Comment:
TODO: add tests in `tests/sdk/test_dict_types.py` to check for discrependiecs between `VaultCreateRequestDict` and `VaultCreateRequest` and all stuff like that